.github/workflows/pr-labeler.yml

```
@@ -0,0 +1,14 @@
+name: "Pull Request Labeler"
```

```suggestion
name: "Pull Request Labeller"
```
Minor grammar thing.

---

Look at you, SWAG hunter ;)